robocop/checkers/__init__.py

```diff
@@ -61,25 +62,25 @@ def configure(self, param, value):
         self.__dict__[param] = value
 
 
-class VisitorChecker(BaseChecker, ast.NodeVisitor):
+class VisitorChecker(BaseChecker, ast.NodeVisitor):  # noqa
     type = 'visitor_checker'
 
-    def visit_File(self, node):
+    def visit_File(self, node):  # noqa
         self.generic_visit(node)
 
 
-class RawFileChecker(BaseChecker):
+class RawFileChecker(BaseChecker):  # noqa
```

Comment:
Why here noqa?

Reply:
I don't remember, something with the class name. Comment out and check it or I can do it in the evening.

Reply:
Ok, I checked and it was my mistake. The error was about class docstring. Removing the comment now...